### ISSUE TITLE
Translucent TB not working in windows 11 (work around)

### ISSUE THREAD

**Issue by PeterWouter** [TranslucentTB/TranslucentTB#672]
Many users of Translucent TB complained about clear tb not working in W 11.
I have found a partial work around for this problem.
The functionality of clear tb is working if you change the display option to dark mode.
Go to personalisation / colours / mode (1st. tab.)
Choose dark and configure Translucent TB to clear.
Now you should see your system icons in white.
Of course this is not the real solution to the problem if you want a light mode for your windows, but at least you can see your system information icons again.
Hope Translucent TB will soon come with an update for a real solution. :))

**Comment by moranrs** [comment]
Me too. Use the normal option on the desktop, as shown below
![image](https://user-images.githubusercontent.com/13774031/197514353-bc0abafd-4c75-4c87-8125-bef819eb2d61.png)
Use the blur option, as shown below
![image](https://user-images.githubusercontent.com/13774031/197514531-0c117ea7-a8ac-46f9-b3b0-e7bea4ccd182.png)
As you can see, the taskbar is grayed out
Use the clear option, as shown below
![image](https://user-images.githubusercontent.com/13774031/197514731-27e4f6a3-80b5-4160-b3d6-f5d51680b8ab.png)
In addition, the "Accent color" configuration is as follows
![image](https://user-images.githubusercontent.com/13774031/197514968-1c67cecf-f6ec-49e1-b77e-bd76ffffcd8b.png)
But on another computer there is no problem with translucentTB

**Comment by sylveon** [comment]
#537

**Comment by garrettford** [comment]
I've got dark mode enabled but it doesn't work. It says "cannot open app package."

**Comment by Stabups** [comment]
Yeah, TB isn't working fully for me neither. It started as soon as I updated to windows 11. I even uninstalled the update. When I try using the blur option a black bar just appears. (For Maximized window). Only clear works, which I'll have to work with for now. on 
![image](https://user-images.githubusercontent.com/96211418/198183451-f749542d-7053-45d1-ac27-ac5ffbf03245.png)
![image](https://user-images.githubusercontent.com/96211418/198183561-27ba9fd6-52e7-42ab-92d3-a831c0c1ff54.png)
top of that I'm getting this line above my taskbar which is infuriating.

**Comment by Leandraxe** [comment]
Until today TTB worked very well for me under Windows 11, but since the last update of W11 nothing works anymore... I liked to have the taskbar totally transparent but now it's broken 😭

**Comment by daman215** [comment]
Yes please fix this the new 11 update mess it up 😔

**Comment by rurumuri** [comment]
Me too. Something was wrong when my Windows 11 Beta updated to build 22623.870
My taskbar can't show the expected effect and even when I click the TranslucentTB's icon,It can't make any respondence
(Maybe Microsoft's update about taskbar on tablet or anything else lead to this problem I guess)
![屏幕截图_20221029_095759](https://user-images.githubusercontent.com/108809647/198764567-e79da114-a745-4b4c-8ef8-f144c4062c19.png)
![fb1029b](https://user-images.githubusercontent.com/108809647/198764643-68813970-f288-4fbc-9928-537f51e873f9.png)
![fb1029a](https://user-images.githubusercontent.com/108809647/198764664-e0cb9826-bacb-460a-9fac-7fcc731b950d.png)

**Comment by looterz** [comment]
I had been using the bleeding edge version to fix the thin gray line on Windows 11, but once my insider preview updated to `10.0.22623.0` Clear no longer works, even with dark mode selected, just a heads up.

**Comment by Stabups** [comment]
Quick temporary fix for the line. You can download RoundedTB from the Microsoft store. It allows you to remove the line. It's what I've been using this and it get's the job done. Open the app, set the Margin to 1, and the corner radius to 0.


![image](https://user-images.githubusercontent.com/96211418/198849821-82748485-8229-49ab-87c6-439409a0e0a7.png)
![image](https://user-images.githubusercontent.com/96211418/198849803-79ebb945-410a-4e59-ae78-8eb18f093e55.png)
![image](https://user-images.githubusercontent.com/96211418/198849838-32d4c029-f9f7-42d9-a888-66c0725dd57b.png)

**Comment by lone-assassin-aj** [comment]
> Quick temporary fix for the line. You can download RoundedTB from the Microsoft store. It allows you to remove the line. It's what I've been using this and it get's the job done. Open the app, set the Margin to 1, and the corner radius to 0.
> 
> ![image](https://user-images.githubusercontent.com/96211418/198849821-82748485-8229-49ab-87c6-439409a0e0a7.png) ![image](https://user-images.githubusercontent.com/96211418/198849803-79ebb945-410a-4e59-ae78-8eb18f093e55.png) ![image](https://user-images.githubusercontent.com/96211418/198849838-32d4c029-f9f7-42d9-a888-66c0725dd57b.png)

This works perfectly well. Only drawback I found on my end was that after each update or restart, Margin sets itself back to 0. That's a slight pain but way more bearable than the overall screwed up Win 11

**Comment by jing-wu-ai** [comment]
我也有这种问题 不能透明化真的很让人上火
![图片](https://user-images.githubusercontent.com/68171027/198858786-57b43161-4a74-4147-8f33-c8a00201bf6e.png)
我怀疑就是图片中的这几次更新出了问题 不过我不能确定是哪一个 只能先全部卸载试一下

**Comment by janborup** [comment]
Got same problem after update, no more Translucent TB is working, even my Small TaskbarSI hack is gone - AGAIN... !
![image](https://user-images.githubusercontent.com/6866443/198901112-61dc5f8c-8edf-4ab8-9137-aeec9731da0a.png)

**Comment by Grenouille-2** [comment]
> 我也有这种问题 不能透明化真的很让人上火 ![图片](https://user-images.githubusercontent.com/68171027/198858786-57b43161-4a74-4147-8f33-c8a00201bf6e.png) 我怀疑就是图片中的这几次更新出了问题 不过我不能确定是哪一个 只能先全部卸载试一下

我比较确定是你图片的第一个10.0.22623.875更新，我就是更新了这个之后用不了translucent tb的，而且这次更新之后任务栏还有别的小变化，比如我把鼠标放在任务栏的QQ图标上，本来应该有悬浮窗的，这次更新完不显示了；还有本来点上箭头后显示的隐藏图标是3列，更新完变成5列了，等等。
I think this issue is caused by the build 10.0.22623.875 update, because my translucent tb went wrong just after this update was installed. And there are also other changes on the taskbar, for example, when I put the cursor on the QQ icon on the taskbar there should be a floating hint, but after this update there is nothing. And there were 3 rows of the hidden icons before and 5 rows after this update, etc.
Hope the translucent TB team can soon fix the problem caused by Microsoft's idiots.

**Comment by Grenouille-2** [comment]
> 

Your dark-mode solution doesn't work either🙁
![Untitled](https://user-images.githubusercontent.com/60314805/199671598-65d36972-6940-48c6-9a88-f16cd56bba00.png)

**Comment by daman215** [comment]
I unenrolled of preview bulids so now it works I guess it only on preview bulids which didn't have to many new features I'm on 22623.857 but not preview build so guys if I was you I will unenrolled also all the new features are still there

**Comment by PsychoPops** [comment]
Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

**Comment by gagisu** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

**Thank you, man!** I solved the problem according to your instructions in 3 minutes!

**Comment by gagisu** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

Sorry, I still have a question... how did you come up with the figures I typed that you listed and what program "STTest" is and how can I get to that program.. I want to learn english and it's not my native language and it's a little harder to understand. Thank you in advance. Thanks in advance.

**Comment by Masterrchieff** [comment]
> > Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.
> 
> Sorry, I still have a question... how did you come up with the figures I typed that you listed and what program "STTest" is and how can I get to that program.. I want to learn english and it's not my native language and it's a little harder to understand. Thank you in advance. Thanks in advance.

[this is why](https://github.com/TranslucentTB/TranslucentTB/issues/537#issuecomment-1053807594)

**Comment by Azukio1994** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

谢谢你，哥们~按照你的方法我也成功让我的任务栏再次变回透明的状态了~

**Comment by PsychoPops** [comment]
Awesome glad to help!

**Comment by redsora** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

its working thank u so much

**Comment by ilitur** [comment]
I'm in beta channel. My build is 22623 and ViveTool doesn't recognize it :-(

**Comment by PsychoPops** [comment]
> I'm in beta channel. My build is 22623 and ViveTool doesn't recognize it :-(

I'm on the same build and in beta channel. I used ViveTool GUI and it works for me. Did you try following my directions above?

**Comment by Masterrchieff** [comment]
> > I'm in beta channel. My build is 22623 and ViveTool doesn't recognize it :-(
> 
> I'm on the same build and in beta channel. I used ViveTool GUI and it works for me. Did you try following my directions above?

When I restart my computer every time, the option in vivetoolGUI will be restored to enabled, do you have the same problem?

**Comment by PsychoPops** [comment]
Weird, no I have not had that problem. Although I have had the problem
where a minor update reverts it back to enabled and then I run Vivetool GUI
again and reboot and it goes back to transparent.

**Comment by ilitur** [comment]
> I'm on the same build and in beta channel. I used ViveTool GUI and it works for me. Did you try following my directions above?

Yes, but viveTool still doesn't show my build in the filtered list :-( 
WTF?

**Comment by ilitur** [comment]
> > 
> 
> Your dark-mode solution doesn't work either🙁 ![Untitled](https://user-images.githubusercontent.com/60314805/199671598-65d36972-6940-48c6-9a88-f16cd56bba00.png)

excuse me for the OT. What software is the one that shows cpu / memory load and network traffic?

**Comment by PsychoPops** [comment]
Don't worry about the build number just follow these directions and it
should work it did for me and several others.

Okay so I managed to figure it out. Download vivetool GUI (its just easier
than the terminal command version if you ask me) I used version 1.62. In
the top left box where it says select build just select the first build it
shows. In my case it showed 25236 vivetool GUI will start loading features.
when its done type in the search box 26008830 that should bring you to
STTest highlighted. Now click on Perform action in vivetool GUI and select
deactivate feature. then it should say disabled next to STTest. Close all
windows and reboot your computer and your task bar should be transparent
again. Thats what worked for me.

**Comment by Setnour6** [comment]
Unfortunately, ViveTool GUI and following the steps did not work for me (the option was already disabled), but RoundedTB worked

**Comment by janborup** [comment]
> Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.

Great Thanks...
That works for me, and much better - my "small taskbar" with the TaskbarSI hack also works again on my Windows 11 22H2 build 22623.891.
Thanks for the tip. 😁👍

**Comment by PsychoPops** [comment]
> > Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.
> 
> Great Thanks... That works for me, and much better - my "small taskbar" with the TaskbarSI hack also works again on my Windows 11 22H2 build 22623.891. Thanks for the tip. 😁👍

Thanks for the TaskbarSI trick! I can only use medium because small cuts off date and time. Is there a trick to getting small to work without cutting things off?

**Comment by LeviAckerman1225** [comment]
> Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.

Thanks👍

**Comment by Enderwaiter** [comment]
> > > 
> > 
> > 
> > Your dark-mode solution doesn't work either🙁 ![Untitled](https://user-images.githubusercontent.com/60314805/199671598-65d36972-6940-48c6-9a88-f16cd56bba00.png)
> 
> excuse me for the OT. What software is the one that shows cpu / memory load and network traffic?

This software is called TrafficMonitor and it can also found in github. Here is the link: https://github.com/zhongyang219/TrafficMonitor

**Comment by ilitur** [comment]
> > > > 
> > > 
> > > 
> > > Your dark-mode solution doesn't work either🙁 ![Untitled](https://user-images.githubusercontent.com/60314805/199671598-65d36972-6940-48c6-9a88-f16cd56bba00.png)
> > 
> > 
> > excuse me for the OT. What software is the one that shows cpu / memory load and network traffic?
> 
> This software is called TrafficMonitor and it can also found in github. Here is the link: https://github.com/zhongyang219/TrafficMonitor

I've found it and installed it. Thank you!

**Comment by Huafalin233** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

solved! you just saved my day.

**Comment by ilitur** [comment]
> Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool 
[...]

now it works!!!
thank you!

**Comment by SoujiKwan** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

it`s working! thank you so much!!

**Comment by Leandraxe** [comment]
> Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows, reboot your computer and your task bar should be transparent again. Thats what worked for me.

It worked for me too! Thanks a lot ❤️
Don't forget to restart your computer ;)

**Comment by ridzwanmidris** [comment]
> Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.

It worked for me too but unfortunately, now, my 2nd monitor taskbar is gone completely lol

**Comment by delmonaco01** [comment]
This works for me:
Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest.
Then install the SmartTaskbar app from the Microsoft Store.
Restart your PC.
And the Clear effect WORKS!

**Comment by Leandraxe** [comment]
Idk why but the last update of W11 reactivate the STTest feature

**Comment by PsychoPops** [comment]
> Idk why but the last update of W11 reactivate the STTest feature

This happens just rerun vivtool gui and run through the instructions I posted and it will work again.

**Comment by PsychoPops** [comment]
> > Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.
> 
> It worked for me too but unfortunately, now, my 2nd monitor taskbar is gone completely lol

I dont have dual monitors so I cant try to emulate what your experiencing. Maybe its just a matter of going into settings and unchecking and rechecking enable toolbar on both monitors?

**Comment by iosahil** [comment]
There is a problem with "STTest" too.
Once "STTest" disabled from ViVeTool GUI & reboot, it works perfectly.
But if one reboots again, the "STTest" gets re-enabled and it has to be disable again using ViVeTool GUI & thus, again restart.
(There wasn't any Windows Update in between)

This new "Foldable Taskbar" is causing me headaches now :(

**Comment by PsychoPops** [comment]
> Idk why but the last update of W11 reactivate the STTest feature

Anytime there is a new update it will reset STTest. Just have to run Vivtool again and you will be fine until the next update.

**Comment by szylmzs** [comment]
is there a plan to repair this bug?

**Comment by sylveon** [comment]
Yes. Closing this in favor of #537 which is the same issue.

**Comment by Fernandodacruz** [comment]
> Não se preocupe com o número da compilação, apenas siga estas instruções e deve funcionar para mim e para vários outros. Ok, então eu consegui descobrir isso. Faça o download da GUI do vivetool (é apenas mais fácil do que a versão do comando do terminal, se você me perguntar) Eu usei a versão 1.62. Na caixa superior esquerda, onde diz selecionar compilação, selecione a primeira compilação exibida. No meu caso, mostrou 25236 GUI vivetool começará a carregar recursos. quando terminar, digite na caixa de pesquisa 26008830 que deve levá-lo ao STTest destacado. Agora clique em Executar ação na GUI do vivetool e selecione desativar o recurso. então deve dizer desativado ao lado de STTest. Feche todas as janelas e reinicie o computador e sua barra de tarefas deve ficar transparente novamente. Isso é o que funcionou para mim.

thanks, it worked here

**Comment by eddienakhoy** [comment]
does not work on 22621.1344. Are there solutions?

**Comment by MrOddin** [comment]
> Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.

Thank you boss! That worked for me

**Comment by eddienakhoy** [comment]
Thanks

вт, 7 мар. 2023 г., 20:59 Pandowisk ***@***.***>:

**Comment by shayisnear** [comment]
trying to solve it on latest build, thing that is supposed to show up wont show up

**Comment by EILLOxX** [comment]
> Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.

Is this what you are saying to download ? I'm i looking at the right one ?
![image](https://user-images.githubusercontent.com/128436412/228319561-cccfda89-219a-46d4-b64f-cf15ba1b9f4f.png)

**Comment by PsychoPops** [comment]
> > Don't worry about the build number just follow these directions and it should work it did for me and several others. Okay so I managed to figure it out. Download vivetool GUI (its just easier than the terminal command version if you ask me) I used version 1.62. In the top left box where it says select build just select the first build it shows. In my case it showed 25236 vivetool GUI will start loading features. when its done type in the search box 26008830 that should bring you to STTest highlighted. Now click on Perform action in vivetool GUI and select deactivate feature. then it should say disabled next to STTest. Close all windows and reboot your computer and your task bar should be transparent again. Thats what worked for me.
> 
> Thank you boss! That worked for me

I want to reiterate what has been said in this forum. If you have wondows version 25284 and you try this workaround you are playing at your own risk. This fix still works for me but I am on  22623. Im in the Dev Channel but release preview. Not sure why I havent hit 25284 but when I do, you best believe I wont try this fix! We have all been warned.

**Comment by CitizenDroid** [comment]
TB works perfect when I disable the STTEST in vivietools problem is it only seems to work for a few days!!! then STTEST is reenabled! is there no way to stop windows reenabling STTEST? does anyone have any idea where that service is or is it called by another name!  have tried to find out something on internet about STTEST and its though it does no exist!!!! only somethig called Test mode!

**Comment by Svencore** [comment]
Can everyone say when the Update comes? Waiting for it for Months 😞

**Comment by CitizenDroid** [comment]
> Can everyone say when the Update comes? Waiting for it for Months 😞

Serioulsy doubting one will come anytime soon if at all as I think its a conflict with a built in windows testing tool that windows whats enabled in order for it do what it was  designed to do! probably not needed for most of us using win11 but knowing MS thay want their crap on 24/7. using VIVETools is the only way to keep enabling it

**Comment by Svencore** [comment]
Is the Vivetool fix safe for the newest Win 11 build?

**Comment by CitizenDroid** [comment]
> Is the Vivetool fix safe for the newest Win 11 build?

It is a safe program and it has ability to enable or disable built in windows features so if you run into a issue just reable it but I very much doubt you will as STTest not a critical feature. you will be fine using it. am using latest windows build and been using for a about a year no issues what so ever but as I said the only thing is it may re-enable its sellf a fews days down the line but just follow the steps to disable STTest again. I used a portable version of TranslucentTB and placed it in the startup folder which seemed to prolong the time but wasn't paying attension so tril and error if youwant to tray that as well as VIVE tools which i would do both.

**Comment by Svencore** [comment]
Okay,thank you. If is there a GUI based Version of Vivetool available, do you have a Link?
(Sorry für my bad English, from Germany 😏)

**Comment by clyezen** [comment]
i've already deactive feature STTest but , after several restart it's going back to enable . make me mad to search again and deactive then restart again :( is there no solution about this problem ? . well if there no solution i'm better to not use it xd

**Comment by CitizenDroid** [comment]
> Okay,thank you. If is there a GUI based Version of Vivetool available, do you have a Link? (Sorry für my bad English, from Germany 😏)

Versuchen Sie es mit diesem Link

https://github.com/thebookisclosed/ViVe/releases/download/v0.3.3/ViVeTool-v0.3.3.zip

**Comment by CitizenDroid** [comment]
Just remember none of these maybe permanent fixes as MS keeps enabling STTest but if you install VIVE tools and keep the program its not such a pain to go through to re-enable it again if it reverts back at least you can have it working till a permanent fix is found. you can try all three methods at same time it hurt doing all tree I have and don't get any issues and I think its prolonging the revert back though that might be just in my head LOL. giver a try it can't hurt.

Part A is for those that want TranslucentTB, and Part B fixes TranslucentTB and allows the Taskbar Size tweak to work again.

Of course you can do both Part A and Part B if desired.


Part A
1. Go here, scroll down a bit and download: TranslucentTB-portable-x64.zip (for 64 bit Windows 11).
2. make a folder called TranslucentTB-portable-x64 in C drive Extract the .zip file to it(assuming C:\ is where Win 11 is installed).
3. Open the folder: C:\TranslucentTB-portable-x64 ...and right click: TranslucentTB.exe and choose: Create shortcut.
4. Right click the shortcut and choose: Copy
5. Navigate to: C:\Users\user name\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup ..and in the Startup folder, right click and "paste" the shortcut. This will cause TranslucentTB to start with Windows.



Part B
1. Go here and download ViVeTool-v0.3.2.zip
2. Create a folder in C:\ called zVIVE
3. Extract the contents of the .zip file you downloaded to: C:\zVIVE
4. Open an elevated (as admin), command prompt.
5. Type: cd C:\zVIVE
6. When the prompt changes to... c:\zVive, type: vivetool /disable /id:26008830
7. Reboot the computer, TranslucentTB and Taskbar resizing, should be working again.

https://github.com/thebookisclosed/ViVe/releases/download/v0.3.3/ViVeTool-v0.3.3.zip 
and heres the portable version in case you want o try and install in startup folder
https://github.com/TranslucentTB/TranslucentTB/releases/download/2022.1/TranslucentTB-portable-x64.zip


Method 3 Windows Registry edit always back up your registry first other wise it can go sideways then your stuck.
If you don’t want to use a third-party tool, then changing a registry key in the Windows registry can also make the Windows 11 taskbar transparent. However, proceed with caution as an incorrect or poorly thought-out registry change comes with the risk of permanently damaging your Windows system. We only recommend this method for experienced Windows users.

Proceed as follows:

Step 1: Open the 'Run' menu with the Windows shortcut [Windows] + [R] and enter the CMD command 'regedit' to open the Windows registry editor.

Step 2: Enter the following path in the registry editor:

HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer\Advanced

Step 3: Firstly, save the registry entry in question by clicking on 'File' and 'Export'. This creates a registry backup for the registry entry you want to change in the location you want.

Step 4: When you are in this folder, right click on an empty space in the right section. Go to 'New' and 'DWORD (32-bit) value'.

Step 5: Now enter 'UseOLEDTaskbarTransparency' as the new string. Double click on the new string and set the value from 0 to 1. Accept the changes with “OK”.

Step 6: Restart the computer. The taskbar will now be transparent

**Comment by Svencore** [comment]
Thank you very much! I will try this when Time permitt's. Do you have tested the Registry fix with the newest Win 11 Version? (Moment 3 Update)

**Comment by CitizenDroid** [comment]
> Thank you very much! I will try this when Time permitt's. Do you have tested the Registry fix with the newest Win 11 Version? (Moment 3 Update)

Yes I have edited my registry as well as using Vive tools and with manually installed portable version of Translucent TB in the start up folder its safe to apply all three options at same time it won't hurt anything and am running latest windows 11 update. also it will always be ok to run these no matter what versiuon of windows 11 update your running the only thing that might happen is the STTest might disable again but just run vive tools again and re-enable thats why you should leave Vive tools installed on your system.

**Comment by Svencore** [comment]
Hi,
I have try only Part B, with installed TB, and it works perfectly with Win 11 Home with the latest Updates from today!!!👍😃Thanks! 
If is there a way to make the STTest disabling permanent?

**Comment by CitizenDroid** [comment]
No the STTest seems to enable after a few days if you do all three options I think it lasted a while longer but its trial and error! you still have to go back and use Vive tools to disable it again I know its a pain but untill a fix is found this is the only solution.

**Comment by Svencore** [comment]
Okay, meanwhile waiting for an official Update for TB.😉

**Comment by Astro-WH** [comment]
Thought I was the only one with this issue. Hope this gets fixed soon :)

**Comment by CitizenDroid** [comment]
> Okay, meanwhile waiting for an official Update for TB.😉



Heres a Fix for now.
Not sure ones coming anytime soon from developers as microsoft have ensured STTest enables on every update or randomly!
As I know for sure Vivetools can work but MS keeps resetting it, I desided to search for a alternative way to sort this problem so I don't have to keep messing around with vivetools every other day its a great tool but its a bit of a pain in the rear to keep having to go back to it every four days or so after realising vivetools comes in a command line version as well as a setup version I thought that one way would be to create a script to keep enabling vivetools at shutdown so here is my workaround.

Before you start I guess you already have a vivetools gui setup version installed. if you do you can test this workaround version is working as stated so open vivetools and ENABLE STTest and make sure it confirmed its ENABLED so that when you follow all the instructions below and reboot your system you should have a working transparent taskbar. other wise you will have to wait and see if it worked over time. all my work around does is keeps disabling STTest via command line at shutdown using group polocies ensuring any updates don't interfere at start up any more. you dont have to uninstall vivetools setup version leave it be having two versions won't effect anything of create any issues  plus you still have a gui version available to you should you want it for anything else as command line version is harder to use as its obviously command lines only and is like a portable version no install needed just drop in!.

download Vivetools cammand line version here, 
https://github.com/thebookisclosed/ViVe/releases/download/v0.3.3/ViVeTool-v0.3.3.zip
and extract its contents to windows/system 32. there should be four files.

open  PowerShell ISE and under "FILE" select "NEW" and just add this line of text below,

ViveTool.exe /disable /id:26008830

and save it to a location of your choice and remember remember its location and save as something like  Vivetool_STTest_Disable_      it should have saved it as a .ps1 file.

now open "GPEDIT.MSC" and go to, 
"Local Computer Policy > Computer Configuration > Windows Settings > Scripts (Startup/Shutdown)."
click on "SHUTDOWN" and click "ADD" now select your saved location and select the your ps1 file and add it now click "APPLY" and click "OK "
Reboot PC to test

thats it hopefully you can say good bye to dealing with this untill a fix comes along if one ever does!.
if you want to double check use vivetools gui version again and ENABLE STTest once again make sure it confirmed its ENABLED and then reboot if taskbars transparent again your good to go.

this is the most sensible solution i can think of and am happy I dont have to deal with it at least for now.

**Comment by Svencore** [comment]
Phuuu, Thank you very much, it is very difficult to understand that in english 😉i will try IT👍

**Comment by Svencore** [comment]
You dont have a batch Data File to Download?

**Comment by CitizenDroid** [comment]
> You dont have a batch Data File to Download?

Sorry dude no batch file needed!. making a script in PowerShell ISE is very easy nothing in my discription is hard you just need it translated! use google tranlation, are you Dutch or Deutsch!

**Comment by CitizenDroid** [comment]
this is all you really need to know.
download Vivetools cammand line version here,
https://github.com/thebookisclosed/ViVe/releases/download/v0.3.3/ViVeTool-v0.3.3.zip
and extract its contents to windows/system 32. there should be four files.

open PowerShell ISE and under "FILE" select "NEW" and just add this line of text below,

ViveTool.exe /disable /id:26008830

and save it to a location of your choice and remember remember its location and save as something like Vivetool_STTest_Disable_ it should have saved it as a .ps1 file.

now open "GPEDIT.MSC" and go to,
"Local Computer Policy > Computer Configuration > Windows Settings > Scripts (Startup/Shutdown)."
click on "SHUTDOWN" and click "ADD" now select your saved location and select the your ps1 file and add it now click "APPLY" and click "OK "
Reboot PC to test

**Comment by Svencore** [comment]
Thanks a lot!!!👍👍👍✌️

**Comment by CitizenDroid** [comment]
> Thanks a lot!!!👍👍👍✌️

keine Ursache :)

**Comment by janborup** [comment]
TranslucentTB does not work with Windows 11 22631.1972
Tried all the ViveTool fixes, even this [CitizenDroid](https://github.com/CitizenDroid) [Startup/Shutdown Script fix](https://github.com/TranslucentTB/TranslucentTB/issues/672#issuecomment-1616747297), but TranslucentTB do not work even the ViveTool says 26008830 is disabled.

Any ideas ?

**Comment by CitizenDroid** [comment]
> TranslucentTB does not work with Windows 11 22631.1972 Tried all the ViveTool fixes, even this [CitizenDroid](https://github.com/CitizenDroid) [Startup/Shutdown Script fix](https://github.com/TranslucentTB/TranslucentTB/issues/672#issuecomment-1616747297), but TranslucentTB do not work even the ViveTool says 26008830 is disabled.
> 
> Any ideas ?

Could be literally anything! first off i would check your windows is ok! sounds like your having a windows issue more than a TTB issue I know TTB  isn't working correctly but never came across anyone who cant get it working with vivetools its just vive tools fix keeps being reset that's why i posted my workaround . use System file checker  to make sure any windows files are not corrupt first and make sure things like transparency is enabled in windows personalisation colours not sure if that does affect TTB but best make sure its on then try my workaround uninstalling your TTB first then install the  portable version in the suggested I.e. startup folder making shorts for necessary directory  like I suggested above and follow them steps also make sure your drivers are up to date especially graphics and windows is up to date by manually checking. anyway if you use my method and portable TTB version installed in correct location yo should be good to go if not you probably have a windows issue.  another option to check is by creating an extar windows profile and log on with that one and set TTB up in that sometimes if you have a windows issue it might not show up in a newly created profile, its worth a try!.

**Comment by qwert2812** [comment]
I'm also having the same problem, windows 11 22621.1848. ViveTool says 26008830 is disabled but still doesn't work after rebooting.

**Comment by CitizenDroid** [comment]
> I'm also having the same problem, windows 11 22621.1848. ViveTool says 26008830 is disabled but still doesn't work after rebooting.

Try installing both versions of TTB and just allow them to run together. first install portable version to what ever location you want then create a short cut to it and paste the short cut  in the startup folder as follows,
C:\Users\usualy you!\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup
Then install the normal setup of TB obtained from github or microsoft store its up to you which one you want to go with but either will do. then once both versionns are installed go back to vivetool start it up with admin rights and disable STTest as discribed. then reboot to see if working.
if you get it to finally run then follow my vivetool workaround in earlier comment to keep it running.  
please post back for others to see what works and what doesn't as many have different issues I can only tell you what worked for me.

**Comment by CitizenDroid** [comment]
Not sure that windows build has anything to do with it not working I have used this on preview builds and on final build and updated regually  with no issues.
I am on win11 pro build 22621.1848 
I can not speak for any other version below pro that might not be able to run TTB like the home version as it has many limits especially graphics! hence why Aero Glass wasn't available in Home versions.

just updated to build 22621.1928 still  working fine

**Comment by Svencore** [comment]
I have the newest Win 11 Home with the latest Updates and it works perfectly. My Win 11 Pro Versions are working with Vivetool and TB , with many Help and exacly do what DAZ says, perfectly too.

**Comment by CitizenDroid** [comment]
> I have the newest Win 11 Home with the latest Updates and it works perfectly. My Win 11 Pro Versions are working with Vivetool and TB , with many Help and exacly do what DAZ says, perfectly too.

nice one for updating situation and for clarifying it works on home version I assumed it wouldn't. glad to hear your sorted. 👍

**Comment by CitizenDroid** [comment]
> I have the newest Win 11 Home with the latest Updates and it works perfectly. My Win 11 Pro Versions are working with Vivetool and TB , with many Help and exacly do what DAZ says, perfectly too.

Hi You may wish to try another little free app that turns employer windows transparent called Glass2K been using it for years its no longer supported but still on offer and works a treat give it a look.
read about it  here .       https://chime.tv/products/glass2k.shtml         
download here.              https://chime.tv/products/glass2k/Glass2k.exe

**Comment by Svencore** [comment]
I will test it next days! Thanks!

**Comment by CitizenDroid** [comment]
> I will test it next days! Thanks!

I think it helps if you make a short cut to glass2k app which needs no install  just drop file where ever you want and make short cut and place it in the windows startup folder like I explained to do with portable TTB. then go to setting and select autoload then save you can also choose your keys as well in settings. it will make any windows semi transparent . its a nice add on.

**Comment by CitizenDroid** [comment]
for any one still stuck I cant remember where I found this info but you could try these fixes below.

Fix 1: Disable STTest using ViveTool
One of the users on GitHub suggested that you disable STTest using ViveTool. Several other users claim that doing this solved the issue for them. Here’s what you need to do:

Download the ViveTool GUI from the GitHub page. Ensure that you download the Windows executable .exe file.
Install ViveTool by following the on-screen instructions.
Open ViveTool and click on the Manually change a Feature button or press F12.
In the text field, enter 26008830 and select the Deactivate Feature option from the drop-down menu next to the text field.
You should see a success message indicating that STTest has been disabled.
Restart your PC and check if TranslucentTB is working fine.

Fix 2: Re-register the Taskbar
If the above solution doesn’t work, try re-registering the taskbar. The solution is to create a batch file on your PC and re-register the taskbar. Here’s how to do it:

Open Notepad and copy the following code into it:
@echo off

taskkill /f /im explorer.exe

taskkill /f /im shellexperiencehost.exe

timeout /t 3 /NOBREAK > nul

del %localappdata%\Packages\Microsoft.Windows.ShellExperienceHost_cw5n1h2txyewy\TempState* /q

timeout /t 2 /NOBREAK > nul

start explorer

@echo on

Click on File and select Save as.
In the Save As window, name the file ClearTileCache.bat.
Then, select the location where you want to save the file and click on Save.
Navigate to the file that you saved in the last step, right-click on it, and select Run as administrator.
While the batch file is in operation, the cache of the start menu will be erased. This may cause the desktop and other elements to disappear for some time.

Fix 3: Restart the Taskbar
If TranslucentTB is still not working properly on your computer, restarting the taskbar may help to resolve the issue. Here are the steps to restart the taskbar:

Press your keyboard’s Ctrl + Shift + Esc keys to open the Task Manager.
Find the “Windows Explorer” process in the Task Manager under the “Processes” tab.
Right-click on “Windows Explorer” and select “Restart” from the context menu.
Wait a few seconds for the taskbar to restart.
Launch TranslucentTB and see if the issue is resolved.
If restarting the taskbar doesn’t fix the issue, you may want to try reinstalling TranslucentTB or checking for any updates to the software.

Fix 4: Repair/Reset TranslucentTB
The third solution to try is to repair or reset TranslucentTB. Sometimes, the issue may be associated with the app itself, so resetting or repairing it should help. Here’s how to do it:

Open the Windows Settings by pressing the Windows key + I on your keyboard.
Navigate to Apps > Installed Apps.
Find the TranslucentTB app and click on the three dots on the far right.
Choose Advanced options.
Under the Reset section, click Repair.
Check if the taskbar is fixed after the repair.
If the taskbar is not fixed, click on the Reset button, which is located below the Repair button.

Fix 5: Update Windows
An outdated version of Windows often results in poor computer performance and many issues, including TranslucentTB not working. 
If you have not updated your Windows software for a long time, do it now. Here’s what you need to do:

Open the Windows Settings by pressing the Windows key + I on your keyboard.
Click on Windows Update from the left panel.
Then, click on Check for updates. Make sure you have an active internet connection during the process.
If any updates are found, click on Download & install or Download & install all. 
Once the update is installed, restart your PC for the changes to take effect.
On the next startup, check if TranslucentTB is now working. If TranslucentTB is still not working, move down to the next solution.

Fix 6: Uninstall Windows Update
While Windows updates are extremely useful for solving issues, they can often cause incompatibility issues with certain apps, and TranslucentTB is no exception. If the issue began after installing a recent Windows update, try uninstalling the update. Here’s what you need to do:

Open the Windows Settings by pressing the Windows key + I on your keyboard.
Click on Windows Update from the left panel.
Now, click on Update history.
Under the Related settings section, click on Uninstall updates.
Find the most recent updates that could be causing the issue and click Uninstall.

Fix 7: Switch to Windows 11 Main Build
Many users have reported that the issue is being experienced predominantly on the preview builds of Windows 11. Microsoft releases preview builds to gather feedback from users and improve the operating system. However, these builds might contain bugs or issues that can cause problems with your computer.

So, if you are on one of the preview builds of Windows 11, and you want to use TranslucentTB, it’s time to switch back to the main build. Here’s what you need to do:

Open the Windows Settings by pressing the Windows key + I on your keyboard.
Click on Windows Update from the left panel.
Click on Windows Insider Program from the right panel.
Then, click on “Stop getting preview builds” on the next page.
Toggle the button next to “Unenroll this device when the next version of Windows releases.” You will now stop getting Windows 11 preview builds.
Then, click on the “Leaving the Insider Program” link at the bottom to proceed.
You will be directed to Microsoft’s webpage. Click on “Leave the program now”, and you will be out of the Windows Insider Program.
Now, go back to the Settings window and click on “Check for updates” to get the latest version of Windows 11.
Your PC will download and install the latest Windows 11 main build version.
Once everything is done, try using TranslucentTB again. It should be working without any issues.

Fix 8: Use Dark Mode
The light theme or light mode can also interfere with the TranslucentTB app. So, it is advisable to use dark mode for the best results with the app. Follow these steps to turn on the dark mode on your Windows 11 PC:

Open the Windows Settings by pressing the Windows key + I on your keyboard.
From the left panel, select Personalization.
Then, click on Colors.
Click on the drop-down next to “Choose your color” and select Dark.
Try using TranslucentTB again, and everything should work smoothly.

Fix 9: Disable Other Customization Tools
If you have other customization tools installed on your system, they may conflict with TranslucentTB and prevent it from working correctly. How to fix this? Simply disable other customization tools and see if TranslucentTB works. Here’s how to disable the other customization tools:

Open the Start menu and search for “Task Manager” or press “Ctrl + Shift + Esc” to open it directly.
In the Task Manager, select the “Startup” tab.
Here, you will see a list of all the programs that start with Windows. Look for any customization tools or applications that might conflict with TranslucentTB, such as other taskbar customizers, system optimizers, or display enhancement tools.
Once you’ve found the conflicting program, right-click on it and select “Disable”. This will prevent the program from starting automatically with Windows.
Alternatively, you can go to the Control Panel and click on “Programs and Features” to see a list of installed programs. Find the customization tool that might conflict with TranslucentTB and uninstall it completely.

After disabling or uninstalling the conflicting program, restart your computer.

Open TranslucentTB again and see if it’s working correctly. If it’s still not working, disable any other customization tools that might conflict with it and repeat the process until TranslucentTB starts working.

Fix 10: Update your Graphics Drivers
In some cases, a corrupt or outdated driver may be why TranslucentTB not working on Windows 11. You can manually check for driver updates or by using a driver update tool. Here’s how to check the driver updates manually:

Right-click on the Start icon or press the Windows key + X on the keyboard and select Device Manager.
Once the Device Manager is open, expand the Display adapters section.
Right-click on your graphics drivers and click on Update driver.
On the pop-up window, click on “Search automatically for drivers” and wait for the drivers to install.
Once installed, restart your PC and check if TranslucentTB is now working.
Fix 11: Disable your Antivirus
The functionality of TranslucentTB may occasionally be affected by the antivirus and firewall software on your PC. If this is the case, try disabling your antivirus temporarily and see if that fixes the issue. Should this succeed, add TranslucentTB to your antivirus and firewall software’s exemption list.

First, you need to identify if your antivirus and firewall software are blocking TranslucentTB. Check if your antivirus and firewall software is blocking TranslucentTB by looking for any error messages or alerts related to TranslucentTB.

If you find any error messages or alerts, disable your antivirus and firewall software temporarily. The steps to disable your antivirus and firewall software may vary depending on the software you’re using. Generally, you can right-click on the antivirus or firewall software icon in the taskbar and select “Disable” or “Turn Off” for a specified period of time.

Once you have disabled your antivirus and firewall software, try running TranslucentTB again and see if it works correctly.

If TranslucentTB starts working after disabling your antivirus and firewall software, you need to add TranslucentTB to your antivirus and firewall software’s exception list. Adding TranslucentTB to the exception list will allow the software to bypass the security checks and run smoothly.

Once you have added TranslucentTB to the exception list, enable your antivirus and firewall software again. Finally, restart your computer to ensure that the changes take effect.

It is worth noting that disabling antivirus and firewall software or performing a system restore may have security implications. Therefore, it is critical to proceed cautiously and only temporarily disable security software to test whether it is causing issues with TranslucentTB.

Fix 12: Reinstall TranslucentTB
If none of the above steps works, try uninstalling and reinstalling TranslucentTB. This will delete all the TranslucentTB files and give it a fresh start. Here’s how to do it:

Open the Control Panel. On the search bar, type Control Panel and click on the first search result.
Click on Programs and Features.
Find TranslucentTB, and click Uninstall. 
Once TranslucentTB is uninstalled, reinstall the latest version from the TranslucentTB GitHub page.
Summing Up
TranslucentTB is a powerful and customizable utility for Windows 11 users who want to enhance the appearance of their taskbar. Its ability to dynamically adjust transparency based on the background colour, support for multi-monitor setups and extensive customization options make it a popular choice among users. However, it’s common to face issues with it, especially if you use Windows 11. If you’re having trouble with TranslucentTB on Windows 11, there are several ways to fix the issue.

You can try disabling the STTest feature using ViveTool, re-registering the taskbar, repairing/resetting TranslucentTB, uninstalling incompatible Windows updates, using Dark mode, or updating drivers.

Hopefully, one of these methods will help you fix the TranslucentTB not working issue on Windows 11 and get your taskbar back to being transparent again.

**Comment by qwert2812** [comment]
Thanks for the help, I reinstall TTB using windows store then update windows and now it's working again.

**Comment by CitizenDroid** [comment]
> Thanks for the help, I reinstall TTB using windows store then update windows and now it's working again.

Nice one and your welcome

**Comment by Svencore** [comment]
I have tested Glass2K, it is working, but i dont need this😉Thanks! (I know this from XP/Vista) 🙃

**Comment by Svencore** [comment]
@CitizenDroid : Your Script workaround for TranslucentTB don't Work, i have Set the Script now as normal and as Powershell in Shut down and Startup both. Testing on three different Devices (Win 11 Home and Pro with newest Updates. I will inform ✌️😉

**Comment by CitizenDroid** [comment]
getting conflicting comments my email says

@CitizenDroid : Your Script workaround for TranslucentTB works fine and it's Easy! 👍👍👍✌️[Svencore](https://github.com/Svencore)  !!

Well if you created a script correctly and added it as directed there's no reason it shouldn't work other than a windows issue!.
Mine has been working fine for about two weeks since creating the script with no sign of reverting back and enabling the pesky STTest.

I will also suggest trying a superb little app called "WPD" or "Privacy dashboard for Windows". whether the fact I have these processes disabled using this app may or may not contribute to why I'm running so fine! it disables multiple[le windows spyware and shuts down loads of unnecessary processes used by MS to track your activities. it can be disabled or enabled at the flick of a switch then reboot to apply. you guys should try it I literally have 98 % 0f its selections disabled and theres alot of them!! and my system is running smooth as can be. no issues what so ever just leave the yellow marked ones active and window update as well as they are needed and if you donate to him you get a great little Steam add on with a few extras, Ive been using it for years now even testing  Dev versions with it before W11 release and had no issues, check it out.

https://wpd.app/  

well worth a look and 100% trusted as I said been using it for years and I'm always running Eset smart security one of the better antivirus out there though a bit pricey but not if you buy a shared licence on ebay and do a system back up as they can't be reinstalled but come at a fraction of the cost again been using that kind of purchase for years no issues well none except my own fault lol.

**Comment by Svencore** [comment]
"Well if you created a script correctly and added it as directed"
I have make this exacly. You Email to me was deleted by GMX,😞😕
I will see If it works with my Skrips. Thank you!

**Comment by CitizenDroid** [comment]
> "Well if you created a script correctly and added it as directed" I have make this exacly. You Email to me was deleted by GMX,😞😕 I will see If it works with my Skrips. Thank you!

If you send me a email address I will send you a copy of my script

**Comment by Svencore** [comment]
Ok, thats finde,My Adress: ***@***.*** von meinem Atari VCS 2600Am 09.07.2023 13:51 schrieb Daz ***@***.***>:

"Well if you created a script correctly and added it as directed" I have make this exacly. You Email to me was deleted by GMX,😞😕 I will see If it works with my Skrips. Thank you!

If you send me a email address I will send you a copy of my script

**Comment by CitizenDroid** [comment]
Here is my script I created and am using,  I have uploaded it to a site it Expires on Jul 16, 2023, 12:58:17 PM GMT+1
all that is on this script is  the following: ViveTool.exe /disable /id:26008830

https://www.transfernow.net/dl/20230709JVyWT7Ki

after downloading above script finish off by completeing below,

now open "GPEDIT.MSC" and go to, 
"Local Computer Policy > Computer Configuration > Windows Settings > Scripts (Startup/Shutdown)."
click on "SHUTDOWN" and click "ADD" now go to your saved script location and select the my ps1 file and add it now click "APPLY" and click "OK "
Reboot PC to test

**Comment by CitizenDroid** [comment]
I also found another thing that may help some out there it is a batch file to "clear  tile cashe and re-register task bar" it may fix some issues some have though it wont fix all well thats just a guess. I have ran it on my system and its absolutely fine.
here is the script  if you just want to create your own I'm sure you know how to create a batch file!
the site wont allow me to upload a batch file as it reads it as a trojan because it kills the task bar but as you can see from the script theres nothing dodgy looking!

paste this below in a text and save as a batch file.

@echo off

taskkill /f /im explorer.exe

taskkill /f /im shellexperiencehost.exe

timeout /t 3 /NOBREAK > nul

del %localappdata%\Packages\Microsoft.Windows.ShellExperienceHost_cw5n1h2txyewy\TempState* /q

timeout /t 2 /NOBREAK > nul

start explorer

@echo on

**Comment by CitizenDroid** [comment]
There are literally hundreds of reasons why some one may have an issue with any thing running on windows it is microsoft after all!  form corrupted system files or applications to missing system processes that may not have installed for one reason or other and many other reasons way too many to list. I like a system that runs the way I want so when I get an issue and can't correct I start again from scratch till I get it working right, I enjoy manipulating windows so re-installing multiple times don't bother me I've liturally done it a million times or more! thats how you learn the hard way :)

**Comment by Svencore** [comment]
Okay, i have a fresh installed Windows 11 Home and make exacly what you do with this skript, i will See the next days because it works or not. Testing the Batch too. Many,many Thanks!

**Comment by CitizenDroid** [comment]
> Okay, i have a fresh installed Windows 11 Home and make exacly what you do with this skript, i will See the next days because it works or not. Testing the Batch too. Many,many Thanks!

Just one thing!!! I thought you said you have it working on a windows 11 Home version!!!!! but now your saying you doin't!!! that surprised me as I would have thought Home has limitations the same as vista and win7 home had! as they could not run Aero's Transparency so I am not sure it will work on home version!!! but thats just a guess.

you can still use windows 7 pro licence keys to install windows 11 pro I have done it many times already there is no cut off point it was a lie. and if you dont have one you can buy a win 11 pro licence off ebay for cheap but I would opt to find a win7 pro licence and install with that.

**Comment by CitizenDroid** [comment]
and for those who aern't ware there is an alternative to TTB use this  program
ClassicShellSetup_4_3_1 and  translucenttaskbar_1_2_by_arkenthera_dausz1z  there absolutely free and worth a try just delete if not happy. I have used them and there fine. I just prefer minimalistic ways less apps the batter.

**Comment by reality-exe** [comment]
> and for those who aern't ware there is an alternative to TTB use this  program
> ClassicShellSetup_4_3_1 and  translucenttaskbar_1_2_by_arkenthera_dausz1z  there absolutely free and worth a try just delete if not happy. I have used them and there fine. I just prefer minimalistic ways less apps the batter.

Or join the discord and look in the announcements channel...

It's fixed...

**Comment by CitizenDroid** [comment]
> > and for those who aern't ware there is an alternative to TTB use this  program
> > ClassicShellSetup_4_3_1 and  translucenttaskbar_1_2_by_arkenthera_dausz1z  there absolutely free and worth a try just delete if not happy. I have used them and there fine. I just prefer minimalistic ways less apps the batter.
> 
> Or join the discord and look in the announcements channel...
> 
> It's fixed...

nice one cheers  :)

**Comment by Svencore** [comment]
The first Post where i saying it works I have Edit, that was wrong. I have 6(!) PC'S (three Notebooks), 2 Notebooks with Win 11 home and the Rest Desktops with W11 Pro and 1 Notebook with Win 11 Pro. I have tested first with Win 11 Pro on Desktop without disabling manually per Vivetool, i have tested the Skipt only, Shut down and the Taskbar not clear, i have disabled SST Test with Vivetool manually and Set the Skript in Both Start and Shut down and next days i will see what Happens. Thanks

**Comment by Svencore** [comment]
> > and for those who aern't ware there is an alternative to TTB use this  program
> > ClassicShellSetup_4_3_1 and  translucenttaskbar_1_2_by_arkenthera_dausz1z  there absolutely free and worth a try just delete if not happy. I have used them and there fine. I just prefer minimalistic ways less apps the batter.
> 
> Or join the discord and look in the announcements channel...
> 
> It's fixed...

Have a Link? I dont have Discord and i dont will have that 😉

**Comment by reality-exe** [comment]
> > > and for those who aern't ware there is an alternative to TTB use this  program
> > > ClassicShellSetup_4_3_1 and  translucenttaskbar_1_2_by_arkenthera_dausz1z  there absolutely free and worth a try just delete if not happy. I have used them and there fine. I just prefer minimalistic ways less apps the batter.
> > 
> > 
> > Or join the discord and look in the announcements channel...
> > It's fixed...
> 
> Have a Link? I dont have Discord and i dont will have that 😉

https://dev.azure.com/sylve0n/TranslucentTB/_build/results?buildId=383&view=artifacts&pathAsName=false&type=publishedArtifacts

Download the portable version. Make sure the original one (I.e the one you downloaded from this repo or from the microsoft store) is uninstalled.

**Comment by Svencore** [comment]
Perfect, i will test it in the next days!👍✌️
The x64 Release or which Version i need?

**Comment by sylveon** [comment]
Fixed in 2023.1 pending release to the store.
